internal/eval/deque_test.go

```diff
@@ -2,14 +2,14 @@ package eval_test
 
 import (
 	"fmt"
+	"gotest.tools/v3/assert"
```

Comment:
not related to this PR. but a line 208 in this file was leading to "unresolved reference" without this change

Reply:
I have fixed the issue, it was due to function DeepEqual on line 208.